com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -20,6 +20,9 @@ namespace Unity.Netcode.Editor.CodeGen
 
     internal sealed class NetworkBehaviourILPP : ILPPInterface
     {
+        private const string k_ExtensionMethodReadName = "ReadValueSafe";
```

Comment:
This is _super_ super nitpicky, but some of the places this variable is being used are actually checking for regular methods and not extension methods. I would just call these `ReadMethodName` and `WriteMethodName`.

Reply:
Good call, I'll make that change